server/php-runner.cpp

```diff
@@ -151,6 +179,39 @@ void PHPScriptBase::init(script_t *script, php_query_data *data_to_set) {
   PHPScriptBase::ml_flag = false;
 }
 
+// asan_stack_unpoison marks the script stack memory as no longer in use,
+// making it possible to do a longjmp up the stack;
+// if ASAN is disabled, this function does nothing
+// use this function right before doing a longjmp
+void PHPScriptBase::asan_stack_unpoison() {
+#if ASAN7_ENABLED
+  __asan_unpoison_memory_region(run_stack, stack_size);
```

Comment:
ASAN says users should use macros instead of functions
![image](https://user-images.githubusercontent.com/28536346/171874297-2e481dd7-11d5-4420-ae18-9970be312384.png)

Reply:
Good point.